spec/order_spec.rb

```diff
@@ -0,0 +1,34 @@
+require_relative '../lib/order'
+require_relative '../lib/menu'
```

Comment:
Do we need to require the `Menu` class here? Could we isolate the unit tests to not use the real menu class?